chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -630,6 +634,7 @@ impl PeerManagerActor {
         edge: &Edge,
         other: &PeerId,
     ) {
+        // FIXME: how to make sure we don't end up in a loop..
```

Comment:
@pompon0 - we should discuss the behaviour in this scenario ("maybe_remove_connected_peer") and whether we need this at all.